src/main/g8/default.properties

```diff
@@ -1,3 +1,6 @@
-name=my-awesome-lambda
+name=Change Me
```

Comment:
It's not a hint on formatting - just using the same value for everywhere that needs changing.